backend/src/test/java/gov/cdc/usds/simplereport/api/PatientManagementTest.java

```diff
@@ -22,14 +24,19 @@
 
   @Test
   void queryPatientWithFacility() throws Exception {
-    Organization org = _orgService.getCurrentOrganization();
-    Facility place = _orgService.getFacilities(org).get(0);
-    _dataFactory.createMinimalPerson(org, place, "Cassandra", null, "Thom", null);
-    _dataFactory.createMinimalPerson(org, null, " Miriana", "Linas", "Luisito", null);
+    final MutableObject<String> facilityId = new MutableObject<>(); // LOL
```

Comment:
that's always a great sign

Reply:
Yeaaaah.... there are like three ways to fix this problem, and I used two of them and liked neither. 😞